aiidalab_widgets_base/utils/loaders.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+from importlib.resources import Package, files
+from pathlib import Path
+
+from IPython.display import Javascript, display
+
+
+def load_css_stylesheet(
+    package: Package | None = None,
+    css_path: str | Path = "",
+    filename: str = "",
+):
```

Comment:
Having both `package` and `css_path` makes this interface suuuuper messy.

How about we instead only have a single parameter of type `Path` that would be a directory with the css files. (maybe `css_dir` would be a better name, since `css_path` does not make it clear that it is not a path to a single CSS filename. 

It would then be the callers responsibility to convert Package to Path if needed.

@edan-bainglass does that make sense?

Reply:
Sure

Reply:
Hmm, on second thought, the only reason I introduced another use case for the API is due to the relocation of the `test.css` stylesheet outside of the package. So, as stated in the comment regarding `DictViewer`, you are welcome to modify the test to simply look at custom styles for the `DictViewer`, at which point, most of the complexity can be removed.

Reply:
I will not have time to dig into into myself at this point, sorry. I think my suggestion makes sense regardless of the motivation imo. To me it makes more sense if the function accepts a Path to CSS dir instead of Package, it makes it more versatile, and it's users are free to decide where they want to put the css files (i.e. not necessarily  inside the module).

Reply:
Understood. I will take a look at implementing it. Thanks 🙂

Reply:
Thanks for bearing with me. Once we merge this I'll release a new alpha version. We can still iterate on it until the final `2.3.0` release.